### DISCUSSION TITLE
cannot create network - followed guide and it just doesn't work...

### DISCUSSION THREAD

**Discussion by bruteForce93** [dec0dOS/zero-ui#210] [Q&A]
Error response from daemon: driver failed programming external connectivity on endpoint zu-controller (560ebca05d7a90e644a074bf66e82a1bcba190eb5a3c9c0df0c86dd30560bdac): Error starting userland proxy: listen udp4 0.0.0.0:9993: bind: address already in use

I can login but pressing on create new network button does absolutely nothing. Can anybody help out?

**Comment by nityam2007** [comment]
I think you are already having zero tier running , make sure to remove it or 

edit the docker-compose.yml 

like in 

zerotier:
    image: zyclonite/zerotier:1.10.6
    container_name: zu-controller
    restart: unless-stopped
    volumes:
      - ./zerotier-one:/var/lib/zerotier-one
    environment:
      - ZT_OVERRIDE_LOCAL_CONF=true
      - ZT_ALLOW_MANAGEMENT_FROM=0.0.0.0/0
    expose:
      - "9993"
    ports:
      - "9993:9993"

here change 9995 
i.e. 

zerotier:
    image: zyclonite/zerotier:1.10.6
    container_name: zu-controller
    restart: unless-stopped
    volumes:
      - ./zerotier-one:/var/lib/zerotier-one
    environment:
      - ZT_OVERRIDE_LOCAL_CONF=true
      - ZT_ALLOW_MANAGEMENT_FROM=0.0.0.0/0
    expose:
      - "9993"
    ports:
      - "9995:9993"

and one more edit at 

environment:
      - ZU_CONTROLLER_ENDPOINT=http://zerotier:9995/
      - ZU_SECURE_HEADERS=false
      - ZU_DEFAULT_USERNAME=admin
and that's it 

It should work after words
If it is what you want, mark it as answered
